lib/AdapterLdap.php

```diff
@@ -340,4 +340,43 @@ public function getMemberStatusByUserAndVo($user, $vo)
         }
         return Member::VALID;
     }
+
+    public function getResourceCapabilities($entityId, $userGroups)
+    {
+        $facility = $this->connector->searchForEntity(
+            $this->ldapBase,
+            '(&(objectClass=perunFacility)(entityID=' . $entityId . '))',
+            ['perunFacilityId']
+        );
+        $facilityId = $facility['perunFacilityId'][0];
+
+        $resources = $this->connector->searchForEntities(
+            $this->ldapBase,
+            '(&(objectClass=perunResource)(perunFacilityDn=perunFacilityId=' . $facilityId . ','
+            . $this->ldapBase . '))',
+            ['capabilities', 'assignedGroupId']
```

Comment:
Extract attribute names as the class constants and replace usage of hardcoded names with these constants.